README.md

```diff
@@ -74,7 +74,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments <a name="acknowledgements"></a>
 
-- The Web App design was inspired by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+- Hat tip to anyone who inspired this project
```

Comment:
Please note that the link provided for your MIT License is invalid, kindly fix this 🙏 
![Screenshot_20230328_124955](https://user-images.githubusercontent.com/44624138/228227904-f2e6a18e-53c1-47ca-a0ec-63a614d4f4b2.jpg)

Reply:
Well noted. Fixing the issue as soon as possible.